manifests/init.pp

```diff
@@ -240,6 +240,31 @@
 #   The RabbitMQ port.
 # @param python_package
 #   Name of the package required by rabbitmqadmin.
+# @param quorum_membership_reconciliation_enabled
+#   Boolean - Enables or disables continuous membership reconciliation.
```

Comment:
So, because the strings docs get generated with the actual type, I think `Boolean` / `Integer` etc. need to be taken out here. Then it'll get added with the correct type (including optional or not) in `Reference.md` and show up twice
<img width="902" alt="image" src="https://github.com/user-attachments/assets/f80d6ab0-0170-4ca4-9171-c87f744704f4" />

Reply:
Makes sense - no big deal - done!